### DISCUSSION TITLE
Generate voice for subtitles

### DISCUSSION THREAD

**Discussion by madey83** [openai/whisper#2236] [Q&A]
Hello,

I would like to ask if it is possible to use whisper to read subtitles (SRT file ) in polish language and create audio?

**Comment by glangford** [comment]
No. Whisper is speech-to-text only. Google something like "text to speech polish subtitles".

**Reply by madey83** [reply]
I have tried Googe in that way, but I could not find something open source that could read SRT file

**Comment by misutoneko** [comment]
Have you considered [piper?](https://github.com/rhasspy/piper)
The code itself is under MIT license but the voice models may have restrictions (and yes, they do have Polish listed).
Idk if there's anything ready-made for handling .srt with a TTS system, so some coding may be required for that.

There are other alternatives I suppose, this just happens to be the one I've tested myself and found it quite decent.

**Reply by madey83** [reply]
Thank you for answer and suggestion.

For sure there should be something available for reading subtitles due to fact I saw lots of AI dubbing done for polish language.

I will test this piper...😁

**Reply by jayavanth** [reply]
@madey83 did you find something?
